verilog/tools/syntax/verilog_syntax.cc

```diff
@@ -84,6 +86,7 @@ ABSL_FLAG(
     "  sv: strict SystemVerilog-2017, with explicit alternate parsing modes\n"
     "  lib: Verilog library map language (LRM Ch. 33)\n");
 
+ABSL_FLAG(bool, machine, false, "Uses JSON for output. Intended to be used as an input for other tools.");
```

Comment:
Suggest calling this flag something like `export_json`

(Note to self: at some point, these "flags" should become proper sub-commands)

Reply:
Done.